api/contacts/contacts.controller.js

```diff
@@ -0,0 +1,75 @@
+const {
+  listContacts,
+  getById,
+  removeContact,
+  addContact,
+  updateContact,
+} = require("../../contacts");
+const Joi = require("joi");
+
+class ContactsController {
+  async getContacts(_, res) {
+    const contacts = await listContacts();
+    return res.status(200).send(contacts);
+  }
+
+  async getContactById(req, res) {
+    const { contactId } = req.params;
+    const contact = await getById(parseInt(contactId));
+    return contact
+      ? res.status(200).send(contact)
+      : res.status(404).send({ message: "Not found" });
+  }
+
+  async createContact(req, res) {
+    const contact = await addContact({ ...req.body });
+    return res.status(201).send(contact);
+  }
+
+  async deleteContact(req, res) {
+    const { contactId } = req.params;
+    const contact = await getById(parseInt(contactId));
+    if (!contact) return res.status(404).send({ message: "Not found" });
+    await removeContact(parseInt(contactId));
```

Comment:
use radix like this 
    ```await removeContact(parseInt(contactId, 10));```


more info: https://stackoverflow.com/questions/6611824/why-do-we-need-to-use-radix